gph/python/ripser_interface.py

```diff
@@ -605,9 +600,10 @@ def ripser_parallel(X, maxdim=1, thresh=np.inf, coeff=2, metric="euclidean",
             )
     else:
         # Only consider strict upper diagonal
```

Comment:
```suggestion
        # Only consider upper diagonal
```
("strict" would mean no diagonal)